src/js/components/Anchor/Anchor.js

```diff
@@ -65,6 +66,7 @@ const Anchor = forwardRef(
         focus={focus}
         hasLabel={label}
         reverse={reverse}
+        target={target}
```

Comment:
have we found a reason why `target` isn't being passed as part of the `...rest`?

Reply:
it is being passed. it's just not showing up on the dom.

Reply:
And why aren't we seeing it on the DOM if it is passed?

Reply:
im not sure why, but ill see what i can find out.

Reply:
Thanks.